operations/network/dns/PowerDNS_Troubleshooting.md

```diff
@@ -0,0 +1,141 @@
+# PowerDNS troubleshooting
+
+## List DNS zone contents
+
+The PowerDNS zone database is populated with data from two sources
+
+* The cray-powerdns-manager service creates the zones and DNS records based on data sourced from the System Layout Service (SLS)
+* The external DNS records are populated by the cray-externaldns-external-dns service using data sourced from Kubernetes annotations and virtual service definitions.
+
+The `cray-powerdns-visualizer` command can be used to view the zone structure that cray-powerdns-manager will create.
+
+```
+ncn-m001:~ # kubectl -n services exec deployment/cray-powerdns-manager -c cray-powerdns-manager -- cray-powerdns-visualizer
+.
+├── 252.10.in-addr.arpa.
+│   ├── [PTR]  5.252.10.in-addr.arpa.
+│   │   └── sw-leaf-002.nmn.wasp.dev.cray.com.
+│   ├── [PTR]  4.252.10.in-addr.arpa.
+│   │   └── sw-leaf-001.nmn.wasp.dev.cray.com.
+│   ├── [PTR]  3.252.10.in-addr.arpa.
+│   │   └── sw-spine-002.nmn.wasp.dev.cray.com.
+│   ├── [PTR]  6.2.252.10.in-addr.arpa.
+│   │   └── pbs_comm_service.nmn.wasp.dev.cray.com.
+│   ├── [PTR]  5.2.252.10.in-addr.arpa.
+│   │   └── pbs_service.nmn.wasp.dev.cray.com.
+│   ├── [PTR]  4.2.252.10.in-addr.arpa.
+│   │   └── slurmdbd_service.nmn.wasp.dev.cray.com.
+│   ├── [PTR]  3.2.252.10.in-addr.arpa.
+│   │   └── slurmctld_service.nmn.wasp.dev.cray.com.
+│   ├── [PTR]  2.2.252.10.in-addr.arpa.
+│   │   └── uai_macvlan_bridge.nmn.wasp.dev.cray.com.
+...
+```
+
+For more information on External DNS and troubleshooting steps see the [External DNS documentation](../external_dns/External_DNS.md)
```

Comment:
Add a comma after steps and a period after the link.

Reply:
Fixed